test/A.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+var eq = require('./shared/utils').eq;
+var R = require('ramda');
+var S = require('..');
+
+describe('A', function() {
+
+  it('is a binary function', function() {
+    eq(typeof S.A, 'function');
+    eq(S.A.length, 2);
+  });
+
+  it('A(f, x) is equivalent to f(x)', function() {
+    eq(S.A(S.inc, 1), 2);
+    eq(R.map(S.A(R.__, 100), [S.inc, Math.sqrt]), [101, 10]);
+  });
+
+  it('is curried', function() {
+    eq(S.A(S.inc).length, 1);
+    eq(S.A(S.inc)(1), 2);
+  });
+
+});
```

Comment:
Regarding spacing:

  - Let's include one empty line after the `'use strict'` pragma.

  - Let's include one empty line between third-party imports and local imports.

  - Let's include two empty lines after the last import.

  - Let's end each file with exactly one trailing `\n`. This script may be useful:

    ```bash
    #!/usr/bin/env bash
    set -e

    # Update the specified files so that each nonempty file ends with
    # exactly one "\n".
    # http://unix.stackexchange.com/q/81685
    for file ; do
      data="$(<"$file")"
      [[ $data == '' ]] || printf '%s\n' "$data" >"$file"
    done
    ```

Bring it all together, we'd get:

```javascript
'use strict';

var R = require('ramda');

var S = require('..');
var eq = require('./shared/utils').eq;


describe('A', function() {

  it('is a binary function', function() {
    eq(typeof S.A, 'function');
    eq(S.A.length, 2);
  });

  it('A(f, x) is equivalent to f(x)', function() {
    eq(S.A(S.inc, 1), 2);
    eq(R.map(S.A(R.__, 100), [S.inc, Math.sqrt]), [101, 10]);
  });

  it('is curried', function() {
    eq(S.A(S.inc).length, 1);
    eq(S.A(S.inc)(1), 2);
  });

});
```